src/Pyz/Zed/Console/ConsoleDependencyProvider.php

```diff
@@ -0,0 +1,144 @@
+use Spryker\Zed\Development\Communication\Console\RemoveGlueBackendIdeAutoCompletionConsole;
+use Spryker\Zed\Development\Communication\Console\RemoveIdeAutoCompletionConsole;
+use Spryker\Zed\Development\Communication\Console\RemoveServiceIdeAutoCompletionConsole;
+use Spryker\Zed\Development\Communication\Console\RemoveZedIdeAutoCompletionConsole;
+use Spryker\Zed\DocumentationGeneratorRestApi\Communication\Console\GenerateRestApiDocumentationConsole;
+use Spryker\Zed\Kernel\Communication\Console\ResolvableClassCacheConsole;
+use Spryker\Zed\Kernel\Container;
+use Spryker\Zed\Log\Communication\Console\DeleteLogFilesConsole;
+use Spryker\Zed\Monitoring\Communication\Plugin\Console\MonitoringConsolePlugin;
+use Spryker\Zed\Propel\Communication\Console\DatabaseDropConsole;
+use Spryker\Zed\Propel\Communication\Console\DatabaseDropTablesConsole;
+use Spryker\Zed\Propel\Communication\Console\DeleteMigrationFilesConsole;
+use Spryker\Zed\Propel\Communication\Console\DeployPreparePropelConsole;
+use Spryker\Zed\Propel\Communication\Console\EntityTransferGeneratorConsole;
+use Spryker\Zed\Propel\Communication\Console\PropelSchemaValidatorConsole;
+use Spryker\Zed\Propel\Communication\Console\PropelSchemaXmlNameValidatorConsole;
+use Spryker\Zed\Propel\Communication\Console\RemoveEntityTransferConsole;
+use Spryker\Zed\Propel\Communication\Plugin\Application\PropelApplicationPlugin;
+use Spryker\Zed\Scheduler\Communication\Console\SchedulerCleanConsole;
+use Spryker\Zed\Scheduler\Communication\Console\SchedulerResumeConsole;
+use Spryker\Zed\Scheduler\Communication\Console\SchedulerSetupConsole;
+use Spryker\Zed\Scheduler\Communication\Console\SchedulerSuspendConsole;
+use Spryker\Zed\Transfer\Communication\Console\DataBuilderGeneratorConsole;
+use Spryker\Zed\Transfer\Communication\Console\RemoveDataBuilderConsole;
+use Spryker\Zed\Transfer\Communication\Console\RemoveTransferConsole;
+use Spryker\Zed\Transfer\Communication\Console\TransferGeneratorConsole;
+use Spryker\Zed\Transfer\Communication\Console\ValidatorConsole;
+use Spryker\Zed\Twig\Communication\Plugin\Application\TwigApplicationPlugin;
+
+/**
+ * @method \Pyz\Zed\Console\ConsoleConfig getConfig()
+ */
+class ConsoleDependencyProvider extends SprykerConsoleDependencyProvider
+{
+    /**
+     * @var string
+     */
+    protected const COMMAND_SEPARATOR = ':';
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return array<\Symfony\Component\Console\Command\Command>
+     */
+    protected function getConsoleCommands(Container $container): array
+    {
+        $commands = [
+            new TransferGeneratorConsole(),
+            new RemoveTransferConsole(),
+            new EntityTransferGeneratorConsole(),
+            new RemoveEntityTransferConsole(),
+
+            new EmptyAllCachesConsole(),
+
+            // Setup commands
+            new DeployPreparePropelConsole(),
+
```

Comment:
`new InitializeDatabaseConsole(),`

Reply:
this class isn't available in this environment (module not installed)
we might want to add it later